pyHSICLasso/hsic_lasso.py

```diff
@@ -78,6 +72,14 @@ def hsic_lasso(X_in, Y_in, y_kernel, x_kernel='Gauss', n_jobs=-1, discarded=0, B
     index = np.arange(n)
     st = 0
     ed = B**2
+
+    # Normalize data
+    if x_kernel == 'Gauss':
+        X_in = X_in / (X_in.std(0) + 10e-20) * np.sqrt(float(B - 1) / B)
+    if y_kernel == "Gauss":
+        Y_in = Y_in / (Y_in.std(1)[:, None] + 10e-20) * np.sqrt(float(B - 1) / B)
+
```

Comment:
Similar to X_in, let's change this as

Y_in = Y_in / (Y_in.std(1)[:, None] + 10e-20)

Please double check the standard deviation of Y_in is 1.